docs/advanced/input_files/input-main.md

```diff
@@ -395,10 +395,19 @@ These variables are used to control general system parameters.
 
 - **Type**: Integer
 - **Description**: takes value 1, 0 or -1.
-  - if set to 1, symmetry analysis will be performed to determine the type of Bravais lattice and associated symmetry operations. (point groups only)
-  - if set to 0, only time reversal symmetry would be considered in symmetry operations, which implied k point and -k point would be treated as a single k point with twice the weight.
-  - if set to -1, no symmetry will be considered.
-- **Default**: 0
+  - 1: No symmetry will be considered.
+  - 0: Only time reversal symmetry would be considered in symmetry operations, which implied k point and -k point would be treated as a single k point with twice the weight.
+  - 1: Symmetry analysis will be performed to determine the type of Bravais lattice and associated symmetry operations. (point groups, space groups, primitive cells, and irreducible k-points)
+- **Default**: 
+  - 0: if *[calculation](#calculation)=md/nscf/istate/ienvelope/get_S*
+  - 1: else
+
+### symmetry_prec
+
+- **Type**: Real
+- **Description**: The accuracy for symmetry judgment.
```

Comment:
please add description for how to set this parameter. which situation? turn up or down?